### PR TITLE
improve: update ui to use api 0.13.0

### PR DESCRIPTION
**Pull request by jmorganca**

### CODE REVIEW

File: ui/components/admin.js [hunk 1]
```diff
@@ -57,43 +57,43 @@ function Grant ({ id, subject, grants }) {
 }
 
 export default function () {
-  const { data: grants } = useSWR(() => '/v1/grants?resource=infra&privilege=admin', { fallbackData: [] })
+  const { data: { items: grants } = {} } = useSWR(() => '/api/grants?resource=infra&privilege=admin', { fallbackData: [] })
```
Comment (jmorganca) [inline ui/components/admin.js:60]: This needs an update to `{ items : [] }`

File: ui/components/grant.js [hunk 2]
```diff
@@ -65,25 +65,25 @@ export default function ({ id }) {
...
+        let res = await fetch(`/api/users?name=${email}`)
         const data = await res.json()
 
         if (!res.ok) {
           throw data
         }
 
-        if (data.length === 0) {
-          res = await fetch('/v1/identities', {
+        if (data?.items?.length === 0) {
```
Comment (hoyyeva) [inline ui/components/grant.js:75]: i think we can do `data?.count`

File: ui/components/grant.js [hunk 3]
```diff
@@ -137,16 +137,16 @@ export default function ({ id }) {
       {error && <ErrorMessage message={error} />}
       {list?.length > 0 &&
```
Comment (hoyyeva) [inline ui/components/grant.js:138]: this should be `list?.count`?

Comment (hoyyeva) [reply ui/components/grant.js:138]: or maybe on the line 24 
``const { data: { items: list } = {} } = useSWR(() => `/api/grants?resource=${destination.name}`)``

it should be 
``const { data: { items: list, count } = {} } = useSWR(() => `/api/grants?resource=${destination.name}`)``

**Review by hoyyeva — APPROVED** [review]

**Pull request merged**